api/src/main/java/org/openmrs/module/appointmentscheduling/AppointmentDailyCount.java

```diff
@@ -0,0 +1,60 @@
+package org.openmrs.module.appointmentscheduling;
+
+import org.openmrs.BaseOpenmrsData;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.List;
+
+public class AppointmentDailyCount extends BaseOpenmrsData implements Serializable {
+
+    private Date date;
+
+    private int dailyCount;
+
+    private List<Appointment> appointments;
+
```

Comment:
I'm not sure why you need the "dailyCount" int on this class if you have the List<Appointment> and Date there.  Wouldn't "getDailyCount()" just return appointments.size()?

Reply:
I want to display the date, the date count and the patient list for each day of the selected month. that's why i'm having the three properties